### ISSUE TITLE
Make bv_maker simpler to start from scratch

### ISSUE THREAD

**Issue by sapetnioc** [brainvisa/brainvisa-cmake#97]
# Issue goal

`bv_maker` command is hard to run for the first time. To setup a build environment, one has to install all required dependencies and then to download and compile a first version of brainvisa-cmake to be able to start the first `bv_maker`. Once the first compilation is done, he can switch to the final version of `bv_maker` (the one that hasd just been built). The goal of this issue is to make it possible to use directly `bv_maker` from source tree without having to build the project a first time. This would not only simplify the building process (removing some rare problems encountered due to the two versions of brainvisa-cmake) but it would also allow to manage sources without brainvisa-cmake. In another context, I am investigating the possibility to download sources using [a project with submodules](https://github.com/sapetnioc/brainvisa-submodules). Therefore I already have all system dependencies and required sources before launching the first `bv_maker`. With this issue, I would like to be able to directly launch `bv_maker` from the downloaded sources to start at the configuration step.

# branvisa-cmake changes

## Make `bv_env` a script instead of a compiled executable

`bv_env` is compiled for compatibility reasons with non Linux systems. Now that we are using a virtual environment for BrainVISA, we are always running it on Linux and can use a script. This will allow to have it ready to use without the need of a first compilation step. `bv_env` (and `bv_env.sh`) would become scripts that would make it possible to use brainvisa-cmake from various context (source code,  build directory, etc.).

## Rename module `brainvisa.maker` to `brainvisa_cmake`

The `maker` module is added to the `brainvisa` module (wich is a directory) as build or install time. This always caused problems to make packages due to the fact that `brainvisa` module must be usable by either brainvisa-cmake and axon even if only one of them is installed. Therefore `brainvisa/__init__.py` file is required in both projects. To date, this file exists only in Axon, therefore brainvisa-cmake Python modules cannot be used by simply make `PYTHONPATH` point to sources. Renaming this module would correct a very old mistake and make brainvisa-cmake python modules truly standard and independant from Axon.

Almost all changes have to be done in brainvisa-cmake. There are three tiny change in casa-distro and web projects.

**Comment by sapetnioc** [comment]
This issue is resolved in the `conda` branch. Modification are now done for compilation in conda environment. I will make another issue.
